xml/security_ldap_replication.xml

```diff
@@ -0,0 +1,645 @@
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Read-write replica</term>
+    <listitem>
+     <para>
+      A replica with a full copy of a database, that accepts read and write
+      operations.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Read-only replica</term>
+    <listitem>
+     <para>
+      A replica with a full copy of a database, that only accepts read
+      operations.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Fractional read-only replica</term>
+    <listitem>
+     <para>
+      A replica with a partial copy of a database, that only accepts read-only
+      operations.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Supplier</term>
+    <listitem>
+     <para>
+      A replica that supplies data from its database to another replica.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Consumer</term>
+    <listitem>
+     <para>
+      A replica that receives data from another replica to write into its
+      database.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Replication agreement</term>
+    <listitem>
+     <para>
+      The configuration of a server defining its supplier and consumer relation
+      to another replica.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Topology</term>
+    <listitem>
+     <para>
+      A set of replicas connected via replication agreements.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Replica ID</term>
+    <listitem>
+     <para>
+      A unique identifier of the &ds389; instance within the replication
+      topology.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>Replication manager</term>
+    <listitem>
+     <para>
+      An account with replication rights in the directory.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect2>
+
+ <sect2 xml:id="sec-security-ldap-replication-configuration">
+  <title>Configuring replication</title>
+  <para>
+   The first example sets up a two node bi-directional replication with a
+   single read-only server, as a minimal starting example. In the following
+   examples, the host names of the two read-write nodes are RW1 and RW2, and
+   the read-only server is RO1. (Of course you must use your own host names.)  
+   </para>
+  <para>
+   All servers should have a backend with an identical suffix. Only one server,
+   RW1, needs an initial copy of the database.
+  </para>
+  <important>
+  <title>Create a strong replication manager password</title>
+  <para>
+    The replication manager should be considered equivalent to the 
+    directory manager, in terms of security and access, and should
+    have a very strong password.
+  </para>  
+  <para>
+    If you create different replication manager passwords for each server,
```

Comment:
Can these be "notes" or "warnings" next to the commands for creating the replication managers?

Reply:
yes!